nim/Dockerfile

```diff
@@ -13,17 +13,7 @@ ENV PATH $PATH:/root/.nimble/bin
 RUN nimble install -y
 
 RUN nim c {{#build_opts}} {{{.}}} {{/build_opts}} \
-    # -d:danger \
```

Comment:
why removing this ?

Reply:
It not works because it commented out.
If this part is necessary for other frameworks, it should be written in each framework itself's config.